github/PullRequest.py

```diff
@@ -448,6 +448,18 @@ def get_review_comments(self):
             None
         )
 
+    def get_single_review_comments(self, id):
+        """
+        :calls: `GET /repos/:owner/:repo/pulls/:number/review/:id/comments <https://developer.github.com/v3/pulls/reviews/>`_
+        :rtype: :class:`github.PaginatedList.PaginatedList` of :class:`github.PullRequestComment.PullRequestComment`
+        """
+        return github.PaginatedList.PaginatedList(
```

Comment:
There should be an `assert` to ensure that `id` is an integer:

```python
assert isinstance(id, (int, long)), id
```

Reply:
The `id` param should also be in the doc string, same as with `get_review_comment` further up in the file.

Reply:
A test for this would also be great, too.

Reply:
1. Fixed the first two comments.
2. I wanted to add a test, but saw that the PR being used in the `PullRequest` tests (#31) doesn't have any reviews. What do you suggest to do?

Reply:
> Fixed the first two comments.

Did you forget to push those changes?

Reply:
@jasonwhite now I pushed the changes :)